.gitignore

```diff
@@ -52,3 +52,4 @@ frontend/coverage
 
 # Logs
 yarn-error.log
+frontend/yarn.lock
```

Comment:
You don't want to ignore this file. This ensures that the exact versions of the packages you intend are installed in production.